pkg/virt-controller/watch/vm.go

```diff
@@ -493,6 +493,43 @@ func (c *VMController) arePVCVolumesReady(vm *virtv1.VirtualMachine) bool {
 	return true
 }
 
+func (c *VMController) hasDataVolumeErrors(vm *virtv1.VirtualMachine) bool {
+	for _, volume := range vm.Spec.Template.Spec.Volumes {
+		if volume.DataVolume == nil {
+			continue
+		}
+
+		dvKey := fmt.Sprintf("%s/%s", vm.Namespace, volume.DataVolume.Name)
+		dvObj, exists, err := c.dataVolumeInformer.GetStore().GetByKey(dvKey)
+		if err != nil {
+			log.Log.Object(vm).Errorf("Error fetching DataVolume %s: %v", dvKey, err)
+			continue
+		}
+		if !exists {
+			log.Log.Object(vm).Warningf("DataVolume %s referenced by VM %s/%s does not exists",
```

Comment:
I think we have such checks and logs already. It may also get quite noise. Can you please ensure that we really need this log message, or at least give it a higher verbosity level?

Reply:
It indeed might get too noisy, and in #6171 we're adding an event+printableStatus for this case. I'll remove that log message.